tests-legacy/PrestaShopBundle/Controller/ControllerTest.php

```diff
@@ -72,6 +88,7 @@ public function itShouldRunTheTestedController($controllerClass)
          * @var Controller $testedController
          */
         $testedController = new $controllerClass();
+        $testedController->setContainer($this->prophesizeSfContainer()->reveal());
```

Comment:
I think it's better to mock $testedController->buildContainer() or $testedController->get() than introducing a public method for testing purposes only, wdyt?